providers/summary/scancode.js

```diff
@@ -66,7 +68,12 @@ class ScanCodeSummarizer {
         if (file.type !== 'file') return null
         const asserted = get(file, 'packages[0].asserted_licenses')
         const fileLicense = asserted || file.licenses
-        const licenses = addArrayToSet(fileLicense, new Set(), license => license.license || license.spdx_license_key)
+        let licenses = new Set(fileLicense.map(x => x.license).filter(x => x))
```

Comment:
what is `licenses[0].license`? I'm not seeing a `license` prop anywhere in the sample data. That would mean that `licenses` here would be empty. Are there ever entries in the `licenses` for a file where that entry is not somehow a license? Seems odd.

Reply:
I left this one from the previous implementation. 

```
const licenses = addArrayToSet(fileLicense, new Set(), license => license.license || license.spdx_license_key)
```

I didn't want to assume I had seen the entire scancode schema and leave this one out